lambdas/gro-convert-to-json/src/main/java/uk/gov/di/data/lep/GroConvertToJson.java

```diff
@@ -146,4 +150,18 @@ private void generateAndAddListOfAuditDataToQueue(List<GroJsonRecordWithHeaders>
             addAuditDataToQueue(auditData);
         }
     }
+
+    @Tracing
+    private String uploadIndividualRegistrations(String bucket, List<GroJsonRecordWithHeaders> deathRegistrations) throws JsonProcessingException {
+        var jsonKeys = new ArrayList<RecordLocation>();
+        for (var registration : deathRegistrations) {
+            var registrationJsonKey = registration.correlationID() + ".json";
+            jsonKeys.add(new RecordLocation(bucket, registrationJsonKey));
+            awsService.putInBucket(bucket, registrationJsonKey, objectMapper.writeValueAsString(registration));
```

Comment:
What happens if one of these fails?

Reply:
In what sense?
They get mapped before and so each registration should be a valid GroJsonRecordWithHeaders - if it isn't then it throws an exception during the converting from xml to json.
If the mapping fails then it throws a JsonProcessing error which gets caught and thrown as a MappingException in the handleRequest function